network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/timeline.html

```diff
@@ -3,7 +3,11 @@
 <section id="recommendation-quote" class="container tw-relative tw-z-10" data-scroll-section>
   <div data-scroll data-scroll-speed="3" data-scroll-sticky data-scroll-target="#recommendation-quote" class="large:tw-mt-[25vh] tw-max-w-3xl mx-auto tw-h-screen tw-flex tw-justify-center tw-items-center tw-text-center tw-flex-col tw-gap-6">
     {% include 'wagtailpages/pages/youtube-regrets-2022/svg/quote.svg' %}
-    <blockquote class="tw-text-[1.75rem] tw-font-zilla">{% trans '“It seems like you routinely have to prune things away or it will keep shoving them in your face until you tell it otherwise.”' %}</blockquote>
+    <blockquote class="tw-text-[1.75rem] tw-font-zilla">{% trans '“It seems like you routinely have to prune things away or it will keep shoving them in your face until you tell it otherwise.”' %}
+      <figcaption class="tw-block">
+        <cite class="tw-not-italic">{% trans '(Participant 112)' %}</cite>
+      </figcaption>
+    </blockquote>
```

Comment:
**Question** Can you give me some more context on why we are wrapping the `cite` in a `figcaption`? According to MDN it seems like `figcaption` is only valid as child of a `figure` element. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption